Searching Algorithms/Jump Search Algorithm/C++/JumpSearch.cpp

```diff
@@ -18,18 +19,21 @@ int jumpSearch(int arr[], int n, int value)
 {
     int i, right, left, jump = (int) sqrt(n);
 
-    // Keep jumping until value is between any two elements of the array or when i goes out of bound
-    for (i = 0; i + jump < n && arr[i] < value; i += jump);
+    for (i = 0; i + jump < n && arr[i] < value; i += jump) {
+        // Keep jumping until value is between any two
+        // elements of the array or when i goes out of bound
+    }
 
-    if (value <= arr[i])    // right is i if search value is less than any jumped element in the array. If not, right is last element
+    if (value <= arr[i]) {
+        // right is i if search value is less than
+        // any jumped element in the array. If not, right is last element
         right = i;
-    else
+    }
+    else {
```

Comment:
`// If not, right is last element` in the previous comment will come under this else, not in the `if` clause. Also please correct the brackets. The code should be -

``` c
    if (value <= arr[i]) {
        // right is i if search value is less than
        // any jumped element in the array.
        right = i;
    } else {
        // If not, right is last element
        right = n;
    }
```